### ISSUE TITLE
Add support for Heroku console

### ISSUE THREAD

**Issue by benjaminoakes** [benjaminoakes/homesick-vi-everywhere#1]
`heroku run console` needs vi mode.  (Didn't it used to work on Bamboo?)

**Comment by hjdivad** [comment]
@ericboehs It's not quite clear to me how your suggestion avoids the complaints you raise.  Whether echoing a string literal or retrieving via `curl`, the problems you raise all stem from writing anything to `inputrc`, no?

**Comment by ericboehs** [comment]
That's correct. My `inputrc` adds back the functionality that creating an `inputrc` removes. I'm recommending creating a similar `inputrc` as echoing all of it proved difficult.

If one's aware of the shortcomings and is adverse to hosting their own `inputrc` online or curling it, then echoing `set editing-mode vi` may be the preferred solution.

**Comment by glittershark** [comment]
Hey all-
I wrote a [buildpack](https://github.com/glittershark/heroku-buildpack-vimode) that uses heroku-buildpack-multi to accomplish this.

**Comment by benlieb** [comment]
This seems to work for me:

.bash_profile
```

function hc { 
  heroku run "bash -i -c 'echo set editing-mode vi > ~/.inputrc'; rails c" 
}

```
